backend/Cache.js

```diff
@@ -0,0 +1,61 @@
+class Cache {
+  constructor(ttl = 3600, maxSize = 100) {
```

Comment:
Seems like a very small limit, considering this cache will be shared by all users?